front/src/applications/stdcm/utils.ts

```diff
@@ -98,14 +136,22 @@ export function getOperationalPointsWithTimes(simulationReport: SimulationReport
       departureMinute
     );
 
+    const duration = getDuration(op.positionOnPath, positions, times);
+    const durationInMinutes = duration !== null ? duration / 1000 : 0;
+    const durationToString = secondsToTimeString(durationInMinutes);
```

Comment:
If you do `/ 1000`, do you rather convert milliseconds to seconds and not to minutes ?

```suggestion
    const durationInSeconds = duration !== null ? duration / 1000 : 0;
    const durationToString = secondsToTimeString(durationInSeconds);
```

Reply:
Nice catch ! done